modules/docker-compose.perception.yaml

```diff
@@ -53,17 +55,6 @@ services:
     image: "${PERCEPTION_TRAFFIC_LIGHT_DETECTION_IMAGE}:${TAG}"
     command: /bin/bash -c "ros2 launch traffic_light_detection traffic_light_detection.launch.py"
 
-  traffic_sign_detection:
-    build:
-      context: ..
-      dockerfile: docker/perception/traffic_sign_detection/traffic_sign_detection.Dockerfile
-      cache_from:
-        - "${PERCEPTION_TRAFFIC_SIGN_DETECTION_IMAGE}:${TAG}"
```

Comment:
please also remove all remnants of PERCEPTION_TRAFFIC_SIGN_DETECTION_IMAGE and its derivatives in the repo